.github/workflows/build.yml

```
@@ -323,21 +323,54 @@ jobs:
       - name: Build bundle and create solution
         run: |
           yarn build:windows
-          node node_modules/react-native-test-app/windows/test-app.js
+          yarn install-windows-test-app --projectDirectory=windows --applicationName=Example
         working-directory: example
       - name: Install NuGet packages
         run: |
           nuget restore
         working-directory: example/windows
       - name: Build
         run: |
-          MSBuild ReactTestApp.sln /t:Rebuild /p:Configuration=${{ matrix.configuration }} /p:Platform=${{ matrix.platform }}
+          MSBuild Example.sln /t:Rebuild /p:Configuration=${{ matrix.configuration }} /p:Platform=${{ matrix.platform }}
         working-directory: example/windows
       - name: Test
         run: |
           MSBuild ReactTestAppTests.vcxproj /t:Build /p:Configuration=${{ matrix.configuration }} /p:Platform=${{ matrix.platform }}
           if ("${{ matrix.platform }}" -eq "x64") { VSTest.Console.exe ${{ matrix.platform }}\${{ matrix.configuration }}\ReactTestAppTests.dll }
         working-directory: example/windows/ReactTestAppTests
+  windows-template:
+    name: "Windows [template]"
+    runs-on: windows-latest
+    strategy: 
+      matrix:
+        configuration: [Debug, Release]
+        platform: [ARM, x64]
```

I don't think we need to build all configurations again for the template builds. The template builds are more focused on whether we can build windows when 1) the user chose all platforms or 2) the user chose Windows only when asked.
```suggestion
        template: [all, windows]
```